lms/djangoapps/grades/subsection_grade.py

```diff
@@ -142,7 +142,8 @@ def attempted_graded(self):
     @property
     def percent_graded(self):
         if self.graded_total.possible > 0:
-            return self.graded_total.earned / self.graded_total.possible
+            # Rounds to two decimal places.
+            return around(self.graded_total.earned / self.graded_total.possible, decimals=2)
         else:
             return 0.0
```

Comment:
Please update this to call `compute_percent` in scores.py - to avoid duplication of the logic.

Reply:
Done.